### ISSUE TITLE
Dapr Installer-Bundle v1.7.4   rumtime is 1.5.3?

### ISSUE THREAD

**Issue by Gsonovb** [dapr/installer-bundle#7]
Hi

I download [[Dapr Installer-Bundle v1.7.4](https://github.com/dapr/installer-bundle/releases/tag/v1.7.4)](https://github.com/dapr/installer-bundle/releases/tag/v1.7.4) and installed, 
but I check version  is   1.5.3  ? why?

```
C:\dapr> dapr --version
CLI version: 1.7.1
Runtime version: 1.5.3
```

605f1873858f110fcf8716760e4f7ef1b08269c5dae24f2c43477caece75bbf4 *daprbundle_windows_amd64.zip

**Comment by msfussell** [comment]
@Gsonovb - we will look into this, since this is clearly wrong.

**Comment by Gsonovb** [comment]
It seems that daprd has changed 1.5.3 to the latest version.

**Comment by msfussell** [comment]
Yes. This is wrong and we will get this fixed ASAP.

**Comment by shivamkm07** [comment]
HI @Gsonovb, Thanks for pointing out the issue. The issue was due to daprd making 1.5.3 as the latest version. For now, we have deleted the 1.7.4 bundle and made 1.7.3 as the latest, since 1.7.4 bundle was clearly incorrect. We will fix the workflow soon and release 1.7.4 correctly

**Comment by shivamkm07** [comment]
This is fixed: Check https://github.com/dapr/installer-bundle/releases/tag/v1.7.4
